test/config/TestConfig.cpp

```diff
@@ -612,8 +927,8 @@ TEST_F(ConfigTestFixture, DeviceDefenderCli)
 {
 	"endpoint": "endpoint value",
 	"cert": "/tmp/aws-iot-device-client-test-file",
+    "root-ca": "/tmp/aws-iot-device-client-test/AmazonRootCA1.pem",
```

Comment:
Some minor formatting problem (same applies a few places below).

Reply:
I think I used tab in the string literal and it caused a diff between the display in my IDE and the string literal.